core/tests/test_utils.py

```diff
@@ -2050,6 +2050,18 @@ def assertRaises(self, exc, fun, *args, **kwds):
             'self.assertRaises should not be used in these tests. Please use '
             'self.assertRaisesRegexp instead.')
 
+    def assertRaisesRegexp(  # pylint:disable=keyword-arg-before-vararg
+            self, expected_exception, expected_regexp, callable_obj=None,
+            *args, **kwargs):
+        if expected_regexp == '':
```

Comment:
Why are we not checking if the expected_exception is null?

Reply:
We don't need that one because in that case expected_regexp won't match expected_exception's message and the test will fail automatically.